roles/test_operator/defaults/main.yml

```diff
@@ -151,6 +151,7 @@ cifmw_test_operator_tobiko_version: master
 cifmw_test_operator_tobiko_pytest_addopts: null
 cifmw_test_operator_tobiko_prevent_create: null
 cifmw_test_operator_tobiko_num_processes: null
+cifmw_test_operator_tobiko_advanced_image_url: "https://softwarefactory-project.io/ubuntu-minimal-customized-enp3s0"
```

Comment:
Care to update the README to point to what's expected from it and the associated value?

Reply:
:facepalm: in a moment